src/radioBrowser.ts

```diff
@@ -267,6 +267,8 @@ export class RadioBrowserApi {
         lastLocalCheckTime: new Date(response.lastlocalchecktime),
         language: response.language.split(','),
         lastCheckTime: new Date(response.lastchecktime),
+        geo_lat: response.geo_lat !== null ? response.geo_lat: undefined,
```

Comment:
You forgot to change the properties to camel case.

Reply:
Sorry